### PR TITLE
Update blogs.html

### LINKED ISSUE #90: Add link of blog to static website
Hi @Dawnellar  👋.
You are to add the link to your blog, on Algorithmia SE's blog to the **[Static Website (Blog Page)](https://alx-se-algorithmia.github.io/blogs.html)** using our method of collaboration, and a pull request.

Blog Title: **Contributing to Open Source with None-code Skills**

See our [CONTRIBUTING.md](https://github.com/ALX-SE-Algorithmia/ALX-SE-Algorithmia.github.io/blob/main/CONTRIBUTING.md) file for more info.
Please notify the @ALX-SE-Algorithmia/code-reviewing-team if you're done.
Thank you, and happy coding!

### PR DESCRIPTION
**Pull request by dohoudaniel**
## Description:
- Fixed a typo error.

## Changes Made:
- Fixed a typo error.

## Related Issue:
- Fixes no issue

## Screenshots (if applicable):
- No screenshots attached.

### CODE REVIEW

File: blogs.html
Comment (Algorithmia-SE-2023) [inline blogs.html:1]: Hi there.
This is a demo of what you might need to change.

**Review by Algorithmia-SE-2023 — CHANGES_REQUESTED** [review]
- Nothing much. This is just a demo.

File: blogs.html
Comment (dohoudaniel) [reply blogs.html:1]: Yes, I know this is just a demo.

Comment (Algorithmia-SE-2023) [reply blogs.html:1]: Weldone, Dev.

**Review by Algorithmia-SE-2023 — APPROVED** [review]

**Comment by Algorithmia-SE-2023** [comment]
Thank you for paying attention to this little detail.
PR merged!

**Pull request merged**